src/orqviz/hessians/hessians.py

```diff
@@ -53,7 +53,10 @@ def get_Hessian(
 
     Args:
         params: Parameter vector at which the Hessian matrix is computed.
-        loss_function: Loss function for which to calculate the Hessian matrix.
+        loss_function: Function for which to calculate the Hessian matrix.
```

Comment:
Perhaps we could create a `LossFunction` type alias to make it cleaner?

Reply:
Sounds good